app/models/form.rb

```diff
@@ -4,10 +4,10 @@ class Form
               :setup_sections, :form_sections, :unresolved_log_redirect_page_id
 
   def initialize(form_path, start_year = "", sections_in_form = [], type = "lettings")
-    if type == "sales"
-      @setup_sections = [Form::Sales::Sections::Setup.new(nil, nil, self)]
+    if type == "sales" || (start_year && start_year.to_i > 2022)
+      @setup_sections = type == "sales" ? [Form::Sales::Sections::Setup.new(nil, nil, self)] : [Form::Lettings::Sections::Setup.new(nil, nil, self)]
```

Comment:
I think if it were me I'd have three conditions here:

```ruby
if type == "sales"
 ..
elsif start_year == 2022

else # .. this one will be depreciated

end
```

I don't think you need to convert `start_year` to an integer because as far as I can see, it's either an empty string, or an integer year.

Reply:
"" > 2022 would fail with argument error, but if we're separating it into 3 ifs that wouldn't matter

Reply:
Ah yes, you're completely right.

Reply:
Ready for review now, but not sure if it makes sense to have 2 ifs, cause the content is very similar